test/components/Modal.test.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Modal component and subcomponent test file
+ * Tests the fullscreen modal component for functionality and content
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+// import Modal from '../../client/src/components/modal/Modal.jsx';
+// import PhotoReel from '../../client/src/components/modal/PhotoReel.jsx';
+
+// import photos from '../../data/testPhotoData.js';
+
+describe('modal component', () => {
+  test('placeholder', () => {});
+});
```

Comment:
Consider pushing this in a later commit when there is more functionality added on to it. Your call!